Source/x11/XGServerWindow.m

```diff
@@ -1893,15 +1894,16 @@ - (int) window: (NSRect)frame : (NSBackingStoreType)type : (unsigned int)style
       _wmAppIcon = -1;
       return win;
     }
-  root = [self _rootWindowForScreen: screen];
-  context = [self xrContextForScreen: screen];
+  root = [self _rootWindow];
+  context = [self screenRContext];
 
   /* Create the window structure and set the style early so we can use it to
   convert frames. */
   window = NSAllocateCollectable(sizeof(gswindow_device_t), NSScannedOption);
   memset(window, '\0', sizeof(gswindow_device_t));
   window->display = dpy;
-  window->screen = screen;
+  window->screen_id = defScreen;
+  window->monitor_id = screen;
```

Comment:
Now you got me completely confused. Is the monitor_id that what in gui is the screen? Wouldn't it be clearer to name them the same, that is keep the screen name but refer to monitors in the XRandR case?

Reply:
> Now you got me completely confused. Is the monitor_id that what in gui is the screen? 

Yes.

>Wouldn't it be clearer to name them the same, that is keep the screen name but refer to monitors in the XRandR case?

screen_id used in multiple places as Xlib screen for Xlib function calls - we should keep that structure element.